.mage/proto.go

```diff
@@ -69,14 +69,14 @@ func makeProtoc() (func(...string) error, *protocContext, error) {
 	return sh.RunCmd("docker", "run",
 			"--rm",
 			"--user", fmt.Sprintf("%s:%s", usr.Uid, usr.Gid),
-			"--mount", fmt.Sprintf("type=bind,src=%s,dst=%s/api", filepath.Join(wd, "api"), wd),
+			"--mount", fmt.Sprintf("type=bind,src=%s,dst=%s/lorawan-stack/api", filepath.Join(wd, "api"), filepath.Dir(wd)),
 			"--mount", fmt.Sprintf("type=bind,src=%s,dst=%s/go.thethings.network/lorawan-stack/pkg/ttnpb", filepath.Join(wd, "pkg", "ttnpb"), protocOut),
-			"--mount", fmt.Sprintf("type=bind,src=%s,dst=%s/sdk/js", filepath.Join(wd, "sdk", "js"), wd),
-			"-w", wd,
+			"--mount", fmt.Sprintf("type=bind,src=%s,dst=%s/lorawan-stack/sdk/js", filepath.Join(wd, "sdk", "js"), filepath.Dir(wd)),
+			"-w", filepath.Join(filepath.Dir(wd), "lorawan-stack"),
 			fmt.Sprintf("%s:%s", protocName, protocVersion),
 			fmt.Sprintf("-I%s", filepath.Dir(wd)),
 		), &protocContext{
-			WorkingDirectory: wd,
+			WorkingDirectory: filepath.Dir(wd),
```

Comment:
```suggestion
			WorkingDirectory: mountWD,
```
And then you can remove all the changes below